.codecov.yml

```
@@ -1,4 +1,7 @@
 coverage:
+  notify:
+    # Keep this in sync with the number of CI jobs uploading coverage.
+    after_n_builds: 3
```

```suggestion
codecov:
  notify:
    # Keep this in sync with the number of CI jobs uploading coverage.
    after_n_builds: 3
coverage:
```
This seems to be valid and I think matches their docs.

---

yep, thanks for fixing! Let's see if it succeeds now.